site/topics/programming/assembler.py

```diff
@@ -100,10 +100,12 @@ def verify_syntax_return_string(program_line):
     raise ValueError(f"Invalid operator and/or operand {program_line}")
 
 
-if len(sys.argv) != 2:
-    raise ValueError(f"Assembler requires exactly one argument specifying a file name, {len(sys.argv) - 1} given")
+if len(sys.argv) != 2 and len(sys.argv) != 3:
```

Comment:
imo make it optional. similar to `gcc`

Reply:
2nd Arg is optional. 

If only 1 , it defaults to a.hex

Reply:
shouldn't this be `or` then? or am I undercaffeinated

Reply:
undercaffeinated. these are `!=`. I was reading `==` (I generally prefer positive conditionals)

anyway